### DISCUSSION TITLE
Is stop bit argument of `I2C` available?

### DISCUSSION THREAD

**Discussion by stc1988** [Moddable-OpenSource/moddable#929] [Q&A] [answered]
I found a [note](https://github.com/Moddable-OpenSource/moddable/blob/public/modules/io/i2c/esp32/_i2c.c#L183) about stop bit argument of `I2C`.

On the other hand, [related issue](https://github.com/espressif/esp-idf/issues/8348) has been closed. Can we use stop bit argument?

**Comment by andycarle** [comment, marked as answer]
@stc1988 Good question. As you note, the underlying issue here with the ESP-IDF has been resolved. To pick up that fix, we need to move the ESP-IDF version to v4.4.2. Fortunately, we are planning on doing that shortly.

I've just committed a fix to re-enable the stop bit argument in the I2C APIs. After our next GitHub push, you should pull the Moddable SDK and also update to ESP-IDF v4.4.2. Then you can use the stop bit argument again.

**Reply by andycarle** [reply]
And that is now live.

**Reply by stc1988** [reply]
Good!! I will try it later.
